datasets/modeled/ind_residuals/dataset.json5

```diff
@@ -0,0 +1,122 @@
+{
+  dataset_id: "decarb_2023_ind_residuals",
+  dataset_type: "modeled",
+  sector_description: "Industrial",
+  description: "Industrial residual load profiles for the contiguous US for DECARB 2023 High Scenario, calculated using industrial and eia861 projected data for 2024.",
+  origin_creator: "Ashreeta Prasanna",
+  origin_organization: "NREL",
+  origin_contributors: [
+    "Ashreeta Prasanna",
+    "Elaine Hale",
+    "Daniel Thom",
+    "Lixi Liu"
+  ],
+  origin_project: "DECARB",
+  origin_date: "2024-07-09",
+  origin_version: "v0.1.0",
+  data_source: "EIA, NREL, IGATE-E",
+  source: "Industrial residual calculation for DECARB 2023",
+  data_classification: "low",
+  tags: [
+    "industrial",
+    "residual"
+  ],
+  trivial_dimensions: [
+    "scenario",
+    "subsector",
+    "weather_year",
+  ],
+  data_schema: {
+    table_format: {
+      format_type: "unpivoted",
+      value_column: "value",
+    },
+    data_schema_type: "one_table",
+  },
+
+  dimensions: [
+    {
+      type: "geography",
+      name: "State",
+      display_name: "State",
+      file: "../../../datasets/modeled/industry/dimensions/states.csv",
+      module: "dsgrid.dimension.standard",
+      class: "State",
+      description: "US States and DC",
+    },
+    {
+      type: "metric",
+      name: "Industrial Residual",
+      display_name: "Industrial residual",
+      file: "dimensions/enduses.csv",
+      module: "dsgrid.dimension.standard",
+      class: "EnergyEndUse",
+      description: "Unspecified end use electricity",
+    },
```

Comment:
Per comment above, should just be "Unspecified Electricity" (or similar, if I've not typed it exactly the same as elsewhere).